docs/topics/Gateway_Events.md

```diff
@@ -543,6 +546,23 @@ Sent when a message is pinned or unpinned in a text channel. This is not sent wh
 | channel_id          | snowflake          | ID of the channel                                       |
 | last_pin_timestamp? | ?ISO8601 timestamp | Time at which the most recent pinned message was pinned |
 
+### Entitlements
+
+#### Entitlement Create
+
+Sent when an entitlement is created. The inner payload is a [entitlement](#DOCS_MONETIZATION_ENTITLEMENTS/entitlement-object) object.
+
+#### Entitlement Update
+
+Sent when an entitlement is renewed for the next billing period. The inner payload is an [entitlement](#DOCS_MONETIZATION_ENTITLEMENTS/entitlement-object) object. The `ends_at` field will have an updated value with the new expiration date.
+
+#### Entitlement Delete
+
+Sent when an entitlement is deleted. The inner payload is a [entitlement](#DOCS_MONETIZATION_ENTITLEMENTS/entitlement-object) object. Entitlements are not deleted when they expire. Entitlement deletions are infrequent and only occur when:
+
+-   Discord issues a refund for a subscription
+-   Discord removes an entitlement from a user via internal tooling
```

Comment:
We should probably not document this as an exhaustive list of reasons why an entitlement may be deleted. It should be sufficient to explain that entitlements are not deleted for subscription renewals/lapses.